Base/usr/share/man/man7/Audio-subsystem.md

```diff
@@ -0,0 +1,60 @@
+## Name
+
+Overview of the SerenityOS audio subsystem, including a brief description of `/dev/audio`, the AudioServer and their interfaces. 
```

Comment:
The reference to `/dev/audio` should probably also be a link?

Reply:
If two links to the same thing in one file are fine, yes